build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     implementation "io.opentelemetry:opentelemetry-exporter-otlp-logs:$otelVersion-alpha"
     implementation "io.opentelemetry:opentelemetry-exporter-logging" // only for debug
     runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-logback-appender-1.0:$otelVersion-alpha")
+    runtimeOnly "io.opentelemetry.instrumentation:opentelemetry-resources:$otelVersion-alpha"
```

Comment:
great finding - didn't know about that!

Just checked out the implementation, and we should probably also use `InetAddress.getLocalHost().getHostName()` instead of `System.getenv("HOSTNAME")` as [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/6b607c1e18e6936f7a7eef46a3be77892cef9b3a/instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/HostResource.java#L29)

Reply:
Good point.